draft-ietf-quic-http.md

```diff
@@ -492,16 +492,28 @@ this header is determined by the stream type.
 ~~~~~~~~~~
 {: #fig-stream-header title="Unidirectional Stream Header"}
 
-Two stream types are defined in this document: control streams
-({{control-streams}}) and push streams ({{server-push}}).  Other stream types
-can be defined by extensions to HTTP/QUIC.
+Some stream types are reserved ({{stream-grease}}).  Two stream types are
+defined in this document: control streams ({{control-streams}}) and push streams
+({{server-push}}).  Other stream types can be defined by extensions to
+HTTP/QUIC.
 
 If the stream header indicates a stream type which is not supported by the
-recipient, this SHOULD be treated as a stream error of type
-HTTP_UNKNOWN_STREAM_TYPE.  The semantics of the remainder of the stream are
+recipient, the remainder of the stream cannot be consumed as the semantics are
 unknown. Implementations SHOULD NOT send stream types the peer is not already
-known to support, since a stream error can be promoted to a connection error at
-the peer's discretion (see {{errors}}).
+known to support. Recipients of unknown stream types MAY trigger a QUIC
+STOP_SENDING frame with an error code of HTTP_UNKNOWN_STREAM_TYPE, but MUST NOT
+consider such streams to be an error of any kind.
```

Comment:
You might want to point out that these cannot include QPACK header blocks unless that is explicitly negotiated.

Reply:
Borrowed the extension language that you MUST NOT send anything that modifies the state of the core protocol, QPACK, or any other extension unless you know the peer supports it.  Do you think that's sufficient?

Reply:
Yeah.